Do not hardcode path to install, install into /sbin, do not hardcode
root user for installing manpages.

--- Makefile.orig	2008-10-25 03:36:32.000000000 +0200
+++ Makefile	2008-10-25 03:37:38.000000000 +0200
@@ -27,10 +27,10 @@ SKFLAGS= $(RPM_OPT_FLAGS) -O3 -DSYSV -fo
 LDFLAGS= -s
 
 # Look where your install program is.
-INSTALL = /usr/bin/install
+INSTALL = install
 
 # Destination paths, set prefix=/opt if required
-BINDIR = $(prefix)/usr/sbin
+BINDIR = $(prefix)/sbin
 MANDIR = $(prefix)/usr/share/man
 
 # There is one report that under an all ELF system there may be a need to
@@ -131,10 +131,10 @@ install_exec: syslogd klogd
 	${INSTALL} -m 500 -s klogd ${BINDIR}/klogd
 
 install_man:
-	${INSTALL} -o ${MAN_USER} -g ${MAN_GROUP} -m ${MAN_PERMS} sysklogd.8 ${MANDIR}/man8/sysklogd.8
-	${INSTALL} -o ${MAN_USER} -g ${MAN_GROUP} -m ${MAN_PERMS} syslogd.8 ${MANDIR}/man8/syslogd.8
-	${INSTALL} -o ${MAN_USER} -g ${MAN_GROUP} -m ${MAN_PERMS} syslog.conf.5 ${MANDIR}/man5/syslog.conf.5
-	${INSTALL} -o ${MAN_USER} -g ${MAN_GROUP} -m ${MAN_PERMS} klogd.8 ${MANDIR}/man8/klogd.8
+	${INSTALL} -m ${MAN_PERMS} sysklogd.8 ${MANDIR}/man8/sysklogd.8
+	${INSTALL} -m ${MAN_PERMS} syslogd.8 ${MANDIR}/man8/syslogd.8
+	${INSTALL} -m ${MAN_PERMS} syslog.conf.5 ${MANDIR}/man5/syslog.conf.5
+	${INSTALL} -m ${MAN_PERMS} klogd.8 ${MANDIR}/man8/klogd.8
 
 obj-m += oops.o
 
